eng/pipelines/templates/stages/archetype-cpp-release.yml

```diff
@@ -139,21 +157,42 @@ stages:
                           
                           Write-Host "git status"
                           git status
-                        displayName: Copy updated files into vcpkg fork for PR
-                        condition: ne(variables['SkipVcpkgUpdate'], 'true')
 
-                      # This template does not need to check the 
-                      # SkipVcpkgUpdate variable, it will stop if there are no
-                      # changes in the git repo.
+                          # Commit changes
+                          Write-Host "git add -A"
+                          git add -A 
+                          Write-Host "git -c user.name=`"azure-sdk`" -c user.email=`"azuresdk@microsoft.com`" commit -m `"Initial vcpkg commit for ${{ artifact.VcpkgPortName }}`""
+                          git -c user.name="azure-sdk" -c user.email="azuresdk@microsoft.com" commit  -m "Initial vcpkg commit for ${{ artifact.VcpkgPortName }}" 
+
+                          # Run vcpkg x-add-version to add version to vcpkg file
+                          Write-Host "./bootstrap-vcpkg.bat"
+                          ./bootstrap-vcpkg.bat
+                          Write-Host "./vcpkg.exe x-add-version ${{ artifact.VcpkgPortName }}"
+                          ./vcpkg.exe x-add-version ${{ artifact.VcpkgPortName }}
```

Comment:
```suggestion
                          Write-Host "./vcpkg.exe x-add-version --overwrite-version ${{ artifact.VcpkgPortName }}"
                          ./vcpkg.exe x-add-version --overwrite-version ${{ artifact.VcpkgPortName }}
```

Reply:
`--overwrite-version` works both when we add the package for the first time, and when it already exists in their repo.
Without that, it would add version only when it is the new package, but have no effect (other than printing a message to console) when doing an update.

Reply:
@antkmsft and I chatted. We narrowed the use of `--overwrite-version` down to attempting to publish the same version twice. That is, if you've already released `azure-core-cpp@1.0.0` and you attempt to release `azure-core-cpp@1.0.0` again (which you should never do) then vcpkg will try to prevent you from doing that. 

We have checks in the code already (based on repo tags) that prevent shipping the same version twice. 

Given the use of `--overwrite-version` I don't think we need this because not having `--overwrite-version` does not stop us from publishing new versions of a package (whether those version sort as before or after already published versions)

Reply:
Per our offline conversation, this is not needed. It is only needed when updating a version that already exists, but we increment version numbers, so there should be no problem. It is actually helpful, because it prevents us from accidentally shipping the same version twice. Thank you Daniel!